monero-outreach-docs/translations/pt-br/quickfacts/QuickFacts_pt-br.md

```diff
@@ -0,0 +1,125 @@
+
+#### SEGURANÇA
+
+Sendo uma criptomoeda descentralizada, o Monero é seguro por uma grande rede de usuários espalhados pelo mundo. Transações são confirmadas por um consenso distrubuído e depois gravadas imutavelmente no _[blockchain](https://www.mycryptopedia.com/what-is-blockchain-technology/)_.
+
+#### ESCALABILIDADE
+
+Diferente do Bitcoin e de outras moedas, as quais costumam ter constantes de código inalterável (_hardcoded constants_), o _[tamanho do bloco](https://www.mycryptopedia.com/block-size-explained/)_ do Monero é adapatado automaticamente ao volume de transações, grantindo menores taxas e confirmações mais rápidas.
+
+#### PRIVACIDADE
+
+Monero usa criptografia sofisticada como _[assinaturas em anel](https://getmonero.org/resources/moneropedia/ringsignatures.html)_, _[transações confidenciais em anel](https://www.mycryptopedia.com/monero-ring-confidential-transactions-ringct/)_, e _[endereços stealth](https://www.mycryptopedia.com/everything-need-know-stealth-addresses/)_ para ofuscar as origens, quantidades e destinatários de todas as transações. Os usuários podem decidir exatamente quais informações e para quem eles querem revelar.
+
+#### RESISTÊNCIA À CENSURA
+
+Dadas as características de privacidade do Monero, nenhum endereço específico ou carteira pode ser proibido por mineradores ou por agentes econômicos. Desta forma, os usuários ficam livres de censura e do _[controle de capital](https://en.wikipedia.org/wiki/Capital_control)_.
+
+#### FUNGIBILIDADE
+
+Monero é _[fungível](https://getmonero.org/resources/moneropedia/fungibility.html)_ porque ele é privado por padrão. Não há histórico visível atrelado a cada moeda. Isso significa que os usuários e os negócios não precisam se preocupar com acusações por terem aceitado dinheiro sujo, e também, que um Monero será sempre igual a outro Monero.
+
+### HISTÓRIA DO MONERO
+
+O Monero foi lançado em Abril de 2014. Foi um lançamento justo e pré-anunciado do código de referência **[CryptoNote](https://cryptonote.org/whitepaper.pdf)**. Não houve mineração prévia nem "instantânea" e nenhuma parcela da recompensa do bloco vai para os desenvolvedores. Veja o tópico original do Bitcointalk **[aqui](https://bitcointalk.org/index.php?topic=563821.0)**.
+
+Monero tem feito grandes avanços desde seu lançamento. Quase todas as melhorias proporcionaram avanços em segurança ou privacidade, ou elas facilitaram seu uso. O Monero continua a desenvolver com os objetivos de privacidade e segurança em primeiro lugar, e facilidade de uso e eficiência em segundo.
+
+### O QUE SIGNIFICA MONERO?
+
+A palavra Monero vem do idioma Esperanto. Os criadores escolheram o uso do Esperanto por este ser um idioma "descentralizado" e que representa a quebra de barreiras entre pessoas em uma escala global. Em Esperanto, a palavra Monero é composta por três elementos livremente dispostos, com uma sílaba cada: mon + er + o. Que significam:
+
+**mon:** dinheiro

```

Comment:
Add some spaces in between like these:
**mon:** dinheiro

**er:** a menor parte

 **o:** uma coisa (gramaticamente falando: um substantivo)